instrumentation/runtime/runtime.go

```diff
@@ -95,6 +95,11 @@ func newConfig(opts ...Option) config {
 	return c
 }
 
+// Name returns the instrumentation scope name.
+func Name() string {
+	return "go.opentelemetry.io/contrib/instrumentation/runtime"
```

Comment:
Would a `const` be better/simpler?

Reply:
Sure. If the proposal passes, I will make the change.

Reply:
If we're going to do this I agree it should be a `const` rather than a function and it needs to be done for all instrumentation.  The name should also be more descriptive, such as `ScopeName`.

Reply:
@utezduyar Please update this PR. I will try to bring this PR during next SIG meeting.